css/styles.css

```diff
@@ -0,0 +1,264 @@
+body {
+  margin: 0;
+  color: #fff;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 15px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.nav-container {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+}
+
+.wrapper {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 44px;
+  bottom: 0;
+  padding: 8px;
+}
+
```

Comment:
- [ ] This section should be styled using flexbox as stated in the requirements

Reply:
I will need clarification here. I used the flexbox CSS in Figma.